Tests/Tracking/migration/DataPipelineMigrationAssistantTests.swift

```diff
@@ -11,17 +11,20 @@ class DataPipelineMigrationAssistantTests: UnitTest {
     // This is a workaround until this code base contains implementation tests. There have been bugs
     // that have gone undiscovered in the code when `CustomerIO` passes a request to `DataPipelineImplementation`.
     private var customerIO: CustomerIO!
-
+    private let profileStoreMock = ProfileStoreMock()
     private let backgroundQueueMock = QueueMock()
+    private let migrationAssistantMock = DataPipelineMigrationMock()
 
     private var migrationAssistant: DataPipelineMigrationAssistant { diGraph.dataPipelineMigrationAssistant }
     private var queueStorage: QueueStorage { diGraph.queueStorage }
 
     override func setUp() {
         super.setUp()
 
+        diGraph.override(value: profileStoreMock, forType: ProfileStore.self)
         diGraph.override(value: dateUtilStub, forType: DateUtil.self)
         diGraph.override(value: backgroundQueueMock, forType: Queue.self)
+        diGraph.override(value: migrationAssistantMock, forType: DataPipelineMigration.self)
```

Comment:
I am curious why this test class is creating an instance of `migrationAssistant` via `diGraph.dataPipelineMigrationAssistant` instead of [what is commonly found in other test classes by using the class constructor](https://github.com/customerio/customerio-ios/blob/56584e39684da0b1ee4d642e43fcbaaee879d213/Tests/Common/Background%20Queue/QueueRunRequestTest.swift#L20-L27).